### PR TITLE
Get the latest image version

### PR DESCRIPTION
**Pull request by punkioudi**
- Get the latest image version from https://github.com/rancher/rancher
- Check if this version exists in Dockerhub
- If yes, the tests run in o.o.o

VR : https://github.com/punkioudi/qe-rancher/actions/runs/621297044

### CODE REVIEW

**Review by ge0r — APPROVED** [review]
Nice addition!

File: .github/workflows/o3_schedule.yaml [hunk 1]
```diff
@@ -6,7 +6,21 @@ on:
...
 jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.3
+      - name: Get most recent image version
+        run: |
+          git clone https://github.com/rancher/rancher.git
+          cd rancher/
+          tag=$(git describe --tag $(git rev-list --tags --max-count=1))
```
Comment (pdostal) [inline .github/workflows/o3_schedule.yaml:17]: So this output is sorted so we're sure that the tag returned is really the latest version?

File: .github/workflows/o3_schedule.yaml [hunk 2]
```diff
@@ -6,7 +6,21 @@ on:
   schedule:
     - cron:  '0 6 * * MON'
 jobs:
+  check:
```
Comment (pdostal) [inline .github/workflows/o3_schedule.yaml]: I'd rather call it `latest_version` or something like that.
But this is minor suggestion of course :)

File: .github/workflows/o3_schedule.yaml [hunk 3]
```diff
@@ -6,7 +6,21 @@ on:
   schedule:
     - cron:  '0 6 * * MON'
```
Comment (pdostal) [inline .github/workflows/o3_schedule.yaml:7]: I'd definitely schedule this multiple times a week! Maybe even twice a day or so :)
But I'm fine to do so later, it's totally up to you.

**Review by pdostal — APPROVED** [review]
Nice! This is very neat!

File: .github/workflows/o3_schedule.yaml (hunk 1 above)
Comment (punkioudi) [reply .github/workflows/o3_schedule.yaml:17]: Yes, exactly!

File: .github/workflows/o3_schedule.yaml (hunk 3 above)
Comment (punkioudi) [reply .github/workflows/o3_schedule.yaml:7]: Yes, this part will not stay as it! I think @ggkioulis is already working on this, so for this PR I will leave it ;)

File: .github/workflows/o3_schedule.yaml (hunk 2 above)
Comment (punkioudi) [reply .github/workflows/o3_schedule.yaml]: done!

File: .github/workflows/o3_schedule.yaml (hunk 3 above)
Comment (pdostal) [reply .github/workflows/o3_schedule.yaml:7]: Gut 👍

**Pull request merged**